ingestfile

```diff
@@ -414,14 +414,12 @@ if [ "${PHASECHECKTEST}" = "Y" ] ; then
     fi
 fi
 
+echo "BUUUUUUUUUUUUTS"
```

Comment:
Thanks this is exactly what I was asking for. ;)

Reply:
lol oops, left in a pro debugging tool